### DISCUSSION TITLE
PocketBase GPT

### DISCUSSION THREAD

**Discussion by orenaksakal** [pocketbase/pocketbase#5596] [Show and tell]
Hey folks 

I'm not the author of this agent but I think most of the questions asked here could be answered by [Pocketbase GPT](https://chatgpt.com/g/g-Owo2FBp4K-pocketbase-gpt) it can assist you and let Gani to take a breath / focus on feature development 💪

Good luck

**Comment by 98mux** [comment]
Create a github bot where it auto-answers all the questions 😜

**Comment by orenaksakal** [comment]
if these were discord threads we could extract all the questions including answers and it would be very useful for repetitive questions

**Reply by tigawanna** [reply]
should be possible to crawl all the discussions with questions marked as answered
